src/libs/PolicyUtils.ts

```diff
@@ -407,6 +411,39 @@ function getSubmitToAccountID(policy: OnyxEntry<Policy>, employeeAccountID: numb
     return getAccountIDsByLogins([employee.submitsTo ?? defaultApprover])[0];
 }
 
+function getSubmitToEmail(policy: OnyxEntry<Policy>, employeeAccountID: number): string {
+    const submitToAccountID = getSubmitToAccountID(policy, employeeAccountID);
+    return getLoginsByAccountIDs([submitToAccountID])[0] ?? '';
+}
+
+function getForwardsToAccount(policy: OnyxEntry<Policy>, employeeEmail: string, reportTotal: number): string {
+    if (!hasAdvancedApprovalEnabled(policy)) {
+        return '';
+    }
+
+    const employee = policy?.employeeList?.[employeeEmail];
+    if (!employee) {
+        return '';
+    }
+
+    const positiveReportTotal = Math.abs(reportTotal);
+    if (employee.approvalLimit && positiveReportTotal > employee.approvalLimit) {
+        return employee.overLimitForwardsTo ?? '';
+    }
```

Comment:
I think we need to add this, because if `employee.overLimitForwardsTo` isn't set, but the `approvalLimit` is set (yes that's weird but could happen in OldDot), we want to still use `employee.forwardsTo`

```suggestion
    if (employee.approvalLimit && employee.overLimitForwardsTo && positiveReportTotal > employee.approvalLimit) {
        return employee.overLimitForwardsTo;
    }
```